### PR TITLE
Update button text for push and num challenge error state

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Resolves: OKTA-366355

## Description:

- Update button text for push and number challenge error state

Screen Shot 2021-02-10 at 9 36 54 AM

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added e2e tests

- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Reviewers:

@gowthamidommety-okta @stevennguyen-okta 

### Issue:

- [ OKTA-366355](https://oktainc.atlassian.net/browse/ OKTA-366355)

### CODE REVIEW

**Review by stevennguyen-okta — CHANGES_REQUESTED** [review]
Minor request: Could you change "Push" and "Notification" to lowercase? This was an error on my part for going out casing standardization. I have corrected the mock.

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -803,7 +803,7 @@ oie.okta_verify.totp.title = Enter a code
 oie.okta_verify.totp.enterCodeText = Enter code from Okta Verify app
 oie.okta_verify.push.title = Get a push notification
 oie.okta_verify.push.sent = Push notification sent
-oie.okta_verify.push.send = Send push notification
```
Comment (gowthamidommety-okta) [inline packages/@okta/i18n/src/properties/login.properties:806]: I vaguely remember we auto push by default.. but i can't remember if that changed. Is there no case where we need `Send push notification`?

**Review by gowthamidommety-okta — COMMENTED** [review]
One question

File: packages/@okta/i18n/src/properties/login.properties (hunk 1 above)
Comment (gowthamidommety-okta) [reply packages/@okta/i18n/src/properties/login.properties:806]: If we don't need `Send push notification` does it make sense to change the key to `oie.okta_verify.push.resend`

Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login.properties:806]: @gowthamidommety-okta  I can update the key name. Currently I only see this key used in one place https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js#L15

Comment (gowthamidommety-okta) [reply packages/@okta/i18n/src/properties/login.properties:806]: got it. If you can change the key name / add a new key that would be great :)

**Review by stevennguyen-okta — APPROVED** [review]
looks good!

**Review by gowthamidommety-okta — APPROVED** [review]
lgtm! One small thing to check before merge: is it ok to delete existing strings?

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -803,7 +803,7 @@ oie.okta_verify.totp.title = Enter a code
...
+oie.okta_verify.push.resend = Resend push notification
```
Comment (pradeepdewda-okta) [inline packages/@okta/i18n/src/properties/login.properties:806]: when we do the i18n translations this key name change gets taken care of ? @evelynteo-okta

**Pull request closed without merging**